nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryFulltextConfig.java

```diff
@@ -51,8 +52,24 @@
 @Features(CoreFeature.class)
 @RepositoryConfig(cleanup = Granularity.METHOD)
 @Deploy("org.nuxeo.runtime.reload")
+@ConditionalIgnoreRule.Ignore(condition = IgnoreNonVCSAndMySQL.class)
 public class TestSQLRepositoryFulltextConfig {
 
+    public static class IgnoreNonVCSAndMySQL implements Condition {
+
+        @Inject
+        protected CoreFeature coreFeature;
+
+        @Override
+        public boolean shouldIgnore() {
+            return coreFeature.getStorageConfiguration().isVCS()
+                    // MySQL fulltext is funky with respect to what words it finds in small databases
+                    // so don't bother testing on MySQL, this is mostly a configuration test anyway
+                    || coreFeature.getStorageConfiguration().isVCSMySQL();
```

Comment:
Here you changed the logic. Previously the test ran if it was on VCS and not on MySQL. Now you ignore if it's VCS or MySQL. That's not the same.